src/app/clusters/media-playback-server/media-playback-server.cpp

```diff
@@ -39,74 +39,158 @@
  ******************************************************************************/
 
 #include <app/Command.h>
+#include <app/clusters/media-playback-server/media-playback-server.h>
+#include <app/common/gen/attribute-id.h>
+#include <app/common/gen/attribute-type.h>
+#include <app/common/gen/cluster-id.h>
+#include <app/common/gen/command-id.h>
+#include <app/common/gen/enums.h>
 #include <app/util/af.h>
 
-bool emberAfMediaPlaybackClusterFastForwardRequestCallback(chip::app::Command * commandObj)
+EmberAfMediaPlaybackStatus mediaPlaybackClusterSendMediaPlaybackRequest(MediaPlaybackRequest mediaPlaybackRequest,
+                                                                        uint64_t deltaPositionMilliseconds);
+
+uint8_t mediaPlaybackClusterPlaybackState;
+
+static void writePlaybackState(chip::EndpointId endpoint, uint8_t playbackState)
+{
+    EmberAfStatus status =
+        emberAfWriteServerAttribute(endpoint, ZCL_MEDIA_PLAYBACK_CLUSTER_ID, ZCL_MEDIA_PLAYBACK_STATE_ATTRIBUTE_ID,
+                                    (uint8_t *) &playbackState, ZCL_INT8U_ATTRIBUTE_TYPE);
+    if (status != EMBER_ZCL_STATUS_SUCCESS)
+    {
+        ChipLogError(Zcl, "Failed to store media playback attribute.");
+    }
+}
+
+static uint8_t readPlaybackStatus(chip::EndpointId endpoint)
 {
-    EmberAfStatus status = EMBER_ZCL_STATUS_SUCCESS;
-    emberAfSendImmediateDefaultResponse(status);
+    uint8_t playbackState;
+    EmberAfStatus status =
+        emberAfReadServerAttribute(endpoint, ZCL_MEDIA_PLAYBACK_CLUSTER_ID, ZCL_MEDIA_PLAYBACK_STATE_ATTRIBUTE_ID,
+                                   (uint8_t *) &playbackState, sizeof(uint8_t));
+    if (status != EMBER_ZCL_STATUS_SUCCESS)
+    {
+        ChipLogError(Zcl, "Failed to read media playback attribute.");
```

Comment:
Shouldn't we set playbackState to something here instead of returning a random value?

Reply:
This one is tricky. If i set it to something "default" here, between available states of 0 and 4 it won't make any sense. 
If I pop to the main function `EMBER_ZCL_MEDIA_PLAYBACK_STATUS_INVALID_STATE_FOR_COMMAND` it doesn't have much sense either as delegate methods should return those states. Meaning I would then have 1 state from the delegate method and 1 state from CHIP not able to write the state. 

I haven't changed anything, If you have any idea what to do here, please let me know. I'll add TODO:

Reply:
Realistically, as long as this attribute is defined on this endpoint the read won't fail.  So in practice if this fails `writePlaybackState` would also fail and hence it doesn't matter whether we call it or not, so it's ok to return a random value here.

Maybe document that if that makes sense?

Reply:
Ok. I'll set some random value and add comment that random value will be returned if there's no playback attribute defined.